### PR TITLE
Add initial page related to documentation design

### PR DESCRIPTION
**Pull request by jdbaldry**

### CODE REVIEW

File: docs/sources/writing-guide/about-documentation-design/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: About our documentation design
+menuTitle: Documentation design
+description: Learn about Grafana's documentation docs pages
+aliases:
+  - /docs/writers-toolkit/latest/writing-guide/about-documentation-design/
```
Comment (jdbaldry) [inline docs/sources/writing-guide/about-documentation-design/index.md:6]:
```suggestion
  - /docs/writers-toolkit/writing-guide/about-documentation-design/
```

There's no latest for Writers' Toolkit

**Review by brendamuir — APPROVED** [review]

File: docs/sources/writing-guide/about-documentation-design/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+weight: 100
+keywords:
+  - Grafana
+  - documentation
+  - page design
+---
+
+# About our documentation design
+
+In late 2022, the documentation website was significantly modified to a more modern design. The goal of the redesign was to make our technical documentation more accessible, modern, and scalable as we grow.
```
Comment (chri2547) [inline docs/sources/writing-guide/about-documentation-design/index.md:16]: @GrafanaWriter , Bits that include dates and words like 'new' and 'now' become stale quickly. Maybe start at "Our docs page take sadvantage of our static site generator, Hugo. As a result, several page elements are automatically managed during the publication ..." - and remove everything before that content?

If you are good with that approach, I can update the PR. Thanks!

Comment (GrafanaWriter) [reply docs/sources/writing-guide/about-documentation-design/index.md:16]: yes, please.

File: docs/sources/writing-guide/about-documentation-design/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  - Grafana
+  - documentation
+  - page design
+---
+
+# About our documentation design
+
+In late 2022, the documentation website was significantly modified to a more modern design. The goal of the redesign was to make our technical documentation more accessible, modern, and scalable as we grow.  
+
+In addition to a new look (including a new typeface and layout), our updated docs pages take better advantage of our static site generator, Hugo. As a result, several elements of the page are now automatically managed during the publication of the page using Hugo's taxonomy, thus the source markdown files **do not need to hand manage** these elements and **do not require** contributors to curate them.
```
[the remark this answers is not in the document]
Comment (GrafanaWriter) [reply docs/sources/writing-guide/about-documentation-design/index.md:18]: Fixing to make more evergreen

File: docs/sources/writing-guide/about-documentation-design/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+The improvements include:
+
+- **Improved navigation.** The left-hand sidebar broadly outlines key topics, with nested related topics underneath. This design supports the philosophy that "every page is page one" and creates an system of documentation around a topic that is easier to reference and navigate.
```
Comment (GrafanaWriter) [inline docs/sources/writing-guide/about-documentation-design/index.md:22]:
```suggestion
- **Navigation to preview primary topics.** The left-hand sidebar broadly outlines key topics, with nested related topics underneath. This design supports the philosophy that "every page is page one" and creates an system of documentation around a topic that is easier to reference and navigate. 
```

File: docs/sources/writing-guide/about-documentation-design/index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+In addition to a new look (including a new typeface and layout), our updated docs pages take better advantage of our static site generator, Hugo. As a result, several elements of the page are now automatically managed during the publication of the page using Hugo's taxonomy, thus the source markdown files **do not need to hand manage** these elements and **do not require** contributors to curate them.
+
+The improvements include:
+
+- **Improved navigation.** The left-hand sidebar broadly outlines key topics, with nested related topics underneath. This design supports the philosophy that "every page is page one" and creates an system of documentation around a topic that is easier to reference and navigate. 
+- **More accessible table of contents.** The table of contents now floats on the page as you scroll to the content that's hidden beneath the fold. You can also view the upcoming topics, to enable a better user experience that helps you navigate to subtopics lower on the page.
```
Comment (GrafanaWriter) [inline docs/sources/writing-guide/about-documentation-design/index.md:23]:
```suggestion
- **Floating table of contents.** The table of contents floats on the page as you scroll to the content that's hidden beneath the fold. You can also view the upcoming topics, to enable a better user experience that helps you navigate to subtopics lower on the page.
```

File: docs/sources/writing-guide/about-documentation-design/index.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+- **Auto-generated _Related documentation_.** Using Hugo's taxonomy, our documentation now automatically finds other documentation that's pertinent to the page you're viewing.
```
Comment (GrafanaWriter) [inline docs/sources/writing-guide/about-documentation-design/index.md:24]:
```suggestion
- **Auto-generated _Related documentation_.** Using Hugo's taxonomy, our documentation automatically finds other documentation that's pertinent to the page you're viewing.
```

File: docs/sources/writing-guide/about-documentation-design/index.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+- **Auto-generated _Related resources from Grafana Labs_.** Hugo's taxonomy again is used to automatically generate this content.
+- **Feedback.** We added more prominent options for feedback from our community.
+
+You can read more about the redesign in our [blog] [https://grafana.com/blog/2023/02/03/grafana-documentation-a-look-at-the-new-and-improved-design/].
```
Comment (GrafanaWriter) [inline docs/sources/writing-guide/about-documentation-design/index.md:28]:
```suggestion
You can read about the redesign of our documentation pages in our [blog] [https://grafana.com/blog/2023/02/03/grafana-documentation-a-look-at-the-new-and-improved-design/].
```

File: docs/sources/writing-guide/about-documentation-design/index.md (hunk 2 above)
Comment (GrafanaWriter) [inline docs/sources/writing-guide/about-documentation-design/index.md:16]:
```suggestion
The documentation website uses a more modern design approach to make our technical documentation more accessible, modern, and scalable.  
```

**Comment by jdbaldry** [comment]
> @jdbaldry - Since you opened the PR, I believe you have to commit my changes :)

Nah, anyone can commit to this branch :)

**Comment by GrafanaWriter** [comment]
@jdbaldry - yep yep. Just committed 'em all :)

File: docs/sources/writing-guide/about-documentation-design/index.md (hunk 1 above)
Comment (GrafanaWriter) [inline docs/sources/writing-guide/about-documentation-design/index.md:6]:
```suggestion
  - 
```

**Comment by jdbaldry** [comment]
I'll leave it to @GrafanaWriter and @chri2547 to decide when to merge this. I'm just a PR opening middleman.

**Review by chri2547 — APPROVED** [review]

File: docs/sources/writing-guide/about-documentation-design/index.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+---
+title: About our documentation design
+menuTitle: Documentation design
+description: Learn about Grafana's documentation docs pages
+aliases:
```
Comment (chri2547) [inline docs/sources/writing-guide/about-documentation-design/index.md:5]:
```suggestion
```

File: docs/sources/writing-guide/about-documentation-design/index.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
+---
+title: About our documentation design
+menuTitle: Documentation design
+description: Learn about Grafana's documentation docs pages
+  -
```
Comment (chri2547) [inline docs/sources/writing-guide/about-documentation-design/index.md:5]:
```suggestion
```

**Pull request merged**